README.md

```diff
@@ -1,8 +1,43 @@
+
+## Skills
+
+**Proficient:** Java, RDBMS/SQL, Agile/Scrum, Linux/Bash, Git, C/C++
+
+**Competent:** Python, R, Docker, PHP, HTML, CSS
+
+**Familiar:** Hadoop, NoSQL, Tableau, Crystal Reports, Scala
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_BraikhnaYousafzai](https://github.com/jarviscanada/jarvis_data_eng_BraikhnaYousafzai)
+
+
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_BraikhnaYousafzai/tree/masterhttps://github.com/jarviscanada/jarvis_data_eng_BraikhnaYousafzai/tree/master/linux_sql)]: Implemented a linux cluster monitoring agent using shell scripts inserting hardware and usage data for each cluster node to a database. The data was collected in real time using a Crontab job and persisted using PostgreSQL. The project was deployed using a docker container and version control was maintained through GitHub.
+
+
+## Highlighted Projects
+**COVID-19 Graphing Program** [[GitHub](https://github.com/bmaqs/CS3307-Assignments/tree/master/A1)]: This project pulls Covid-19 tracking data from an API and sorts the data in ascending or descending order depending on user input. It graphs the data to show various trends such as countries with highest number of total cases or countries with highest number of deaths.
+
+**COVID 19 Contact Tracing System** [[GitHub](https://github.com/bmaqs/CS3307-Assignments/tree/master/Group-45-Project)]: Created a software allowing businesses to track incidences of COVID-19 in their establishments and automates notifications to their customer base of any possible exposures. It also allows consumers to easily report COVID-19 cases they have experienced as well as easily receive notifications about potential exposures.
+
+
+## Professional Experiences
+
+**Software Developer, Jarvis (2020-present)**: Worked on development and engineering projects in an agile enviornment. Worked on a scrum team, including taking on duties as a scrum master.
+
+**Data Analyst, First Canadian Title (2017-2019)**: Collaborated with the project team in the design, implementation, and deployment of a new vendor software. Administrated and managed the vendor database. Built a reporting suite using SQL to visualize vendor performance and client KPI's. Worked as a vendor advisor and client solution consultant finding innovative strategies to achieve performance and financial goals.
```

Comment:
Collaborated with the project team in the design, implementation, and deployment of a new vendor software. Administrated and managed the vendor database. Built a reporting suite using SQL to visualize vendor performance and client KPI's. Worked as a vendor advisor and client solution consultant finding innovative strategies to achieve performance and financial goals.